docs/_components/vitepress/NotFound.vue

```diff
@@ -58,12 +67,29 @@ onMounted(() => {
 </script>
 
 <template>
-  <!-- wkande: May 2023: Added showMsg next line -->
+  <!-- Added showMsg next line -->
   <div class="NotFound" v-if="showMsg === 1">
     <p class="code">404</p>
     <h1 class="title">PAGE NOT FOUND</h1>
     <div class="divider" />
-    <blockquote class="quote">
+
+    <!-- GO TO LEGACY DOCS -->
+    <blockquote class="quote" v-if="legacyURL">
+      <hr />
+      The URL send appears to be a page from the legacy documentation. The
```

Comment:
I'm not sure what "send" means here. Perhaps "The URL you are requesting appears to a page belonging to the legacy documentation."

Reply:
Typo **send** > **sent**. But now I like:

"The page you are requesting appears to 
be in the legacy documentation. The
      button below should take you there."